languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -168202,6 +168202,20 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
u|Conflas Sainte Honorine|[ÉE]lancourt|Guyancourt|Houilles|La Celle Saint Cloud|Le Chesnay|Le Pecq|Le V[ée]sinet|Les Clayes sous Bois|Les Mureaux|Maison Laffitte|Limay|Mantes la Jolie|Mantes la Ville|Marly le Roi|Maurepas|Montigny le Bretonneux|Plaisir|Poissy|Rambouillet|Saint Germain en Laye|Satrouville|Trappes|V[ée]lizy Villacoublay|Versailles|Viroflay)|( Bresuire|Maul[ée]on|Partenay|Niort|Thouars)|( Abbeville|Amiens)|( Albi|Castres|Gaillac|Graulhet|Marzamet)|( Castelsarrasin|Caussade|Moissac|Montauban)|( Brignoles|Draguignan|Fr[ée]jus|Hy[èe]res|La Crau|La Garde|La Seyne sur Mer|La Vilette du Var|Saint Rapha[ëe]l|Six Fours le Plages|Toulon)|( Avignon|Carpentras|Cavaillon|Orange|Pertuis|Sorgues)|( Challans|Fontenay le Comte|La Roche sur Yon|Les Herbies|Les Sables d['´‘’′]Olonne)|( Bruxerolles|Ch[âa]tellerault|Chauvigny|Loudun|Poitiers)|( Limoges|Panazol|Saint Julien)|( [ÉE]pinal|Saint Di[ée])|( Auxerre|Avallon|Joigny|Sens)|( Beaucourt|Belfort|Delle)|( Athis Mons|Br[ée]tigny sur Orge|Brunoy|Bures sur Yvette|Chilly Mazarin|Corbeil Essonnes|Courcouronnes|Dourdan|Draveil|[ÉE]tames|[ÉE]vry|Gif sur Yvette|Grigny|Juvisy sur Orge|Igny|Les Ulis|Longjumeau|Massy|Mennecy|Montgeron|Morangis|Morsang sur Orge|Orsay|Palaiseau|Ris Orangis|Saint Genevi[èe]ve des Bois|Saint Michel sur Orge|Savigny sur Orge|Verri[èe]res le Buisson|Villebon sur Yvette|Vigneux sur Seine|Viry Ch[âa]tillon|Yerres)|( Antony|Asnières sur Seine|Bagneux|Bois Colombes|Boulogne Billancourt|Bourg la Reine|Ch[âa]tenay Malabry|Ch[âa]tillonClamart|Chaville|Clichy|Colombes|Courbevoie|Fontenay aux Roses|Garches|Gennevilliers|Issy les Moulineaux|La Garenne Colombes|Le Plessis Robinson|Levallois Perret|Malakoff|Meudon|Monrouge|Nanterre|Neuilly sur Seine|Puteaux|Rueil Malmaison|Sceaux|Sèvres|Suresnes|Villeneuve la Garenne)|( Aubervilliers|Aulnay sous Bois|Bagnolet|Bobigny|Bondy|Clichy sous Bois|Drancy|[ÉE]pinay sur Seine|La Courneuve|Le Blanc Mesnil|Gagny|Les Lilas|Livry Gargan|Montfermeil|Montreuil|Neuilly sur Marne|Noisy le Grand|Noisy le Sec|Pantin|Pierrefitte sur Seine|Romainville|Rosny sous Bois|Saint Denis|Saint Ouen|Sevran|Stains|Tremblay en France|Villemomble|Villepinte)|( Alfortville|Champigny sur Marne|Choisy le Roi|Cr[ée]teil|Fontenay sous Bois|Ivry sur Seine|Maisons Alfort|Saint Maur des Foss[ée]s|Villejuif|Vincennes|Vitry sur Seine)|( Argenteuil|Bezons|Cergy|Ermont|Franconville|Garge l[èe]s Gonesse|Gonesse|Goussainville|Herblay|Pontoise|Sannois|Sarcelles|Taverny|Villier le Bel)|( Baie Mahault|Capesterare Belle Eau|Lamentin|Le Gosier|Le Moule|Les Abymes|Morne [àa] l['´‘’′]eau|Petit Bourg|Point [àa] Pitre|Saint Martin|Sainte Anne|Sainte Rose)|( Ducos|Fort de France|Le Fran[çc]ois|Le Lamentin|Le Robert|Saint Joseph|Rivi[èè]re Pilote|Sainte Marie|Schoelcher)|( Cayenne|Kourou|Mana|Matoury|Remire Montjoly|Saint Laurent du Maroni)|( La Possession|Le Port|Le Tampon|Saint Andr[ée]|Saint Beno[iî]t|Saint Denis|Saint Joseph|Saint Leu|Saint Louis|Saint Paul|Saint Pierre|Sainte Marie|Sainte Suzanne))
```

Comment:
Maybe it makes sense to create an entity "villes" so that it can be easily used also for other rules? And you can easily add other city names to the entitiy.
@GillouLT